aries-site/src/pages/components/tag.mdx

```diff
@@ -1,144 +1,122 @@
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/AppContainer.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagResults.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/utils.js',
-  ]}
-  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tag/CreateTagSimple.js"
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTagSimple />
-</Example>
+### Tag 
+- Contains simple, concise information used to annotate, categorize, group or otherwise label items.
+- Used to display secondary or supplementary information related to an object or item. 
+- Can be used to communicate the state or condition of something (e.g. published, deployed, new, updated, etc.). State is defined [here](/components/notification#taxonomy).
+- A tag should not be used to convey system status, that is considered primary information.
 
-### Assigning name-value tags
+### Static 
+- All points as above, but static tags are not interactive in any way.
 
-<Example
-  code={[
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/CreateTag.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/AppContainer.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagPageHeader.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/TagResults.js',
-    'https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tag/CreateTag/utils.js',
-  ]}
-  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tag/CreateTag"
-  docs="https://v2.grommet.io/tag?theme=hpe#props"
-  showResponsiveControls={['laptop', 'mobile']}
-  template
->
-  <CreateTag />
-</Example>
+### Interactive 
+- Interactive tags enable users to discover, filter, or navigate to identically tagged items.
+- An interactive tag can be actioned by mouse or keyboard.
 
+### Removable
 
-## Filtering with tags
+- A removable tag is interactive via the icon button.
+- Can be used to dismiss a category/value set from a filtered view.
+- Can be used to remove a value that has been assigned to an item or resource.
 
-Filtering with tags allows the user to search, select, and remove tags from a set of filter criteria in order to find relevant data.
+### Highlight
```

Comment:
On these bullets, it feels repetitive having "A highlight tag.." can we update to just be

- Used to indicate...
- Draws attention...

Reply:
done